src/main/java/net/snowflake/client/core/SFStatement.java

```diff
@@ -649,8 +650,8 @@ public long getConservativeMemoryLimit() {
    */
   @Override
   public String[] getChildQueryIds(String queryID) throws SQLException {
-    QueryStatus qs = session.getQueryStatus(queryID);
-    if (QueryStatus.isStillRunning(qs)) {
+    QueryStatusV2 qs = session.getQueryStatusV2(queryID);
```

Comment:
Do we really need to change fro older version of call to v2 version here?

Reply:
The old version is marked as deprecated and shouldn't be used. In my opinion it is better to not use it internally as well, as this is error-prone - even if now we use it correctly, someone unaware may use it incorrectly in the future.